config/index.js

```diff
@@ -623,6 +623,12 @@ var conf = convict({
       format: Boolean,
       env: 'SMS_ENABLED'
     },
+    useMock: {
+      doc: 'Use a mock SMS provider implementation, for functional testing',
+      default: false,
+      format: Boolean,
+      env: 'SMS_USE_MOCK'
```

Comment:
Another way to kind of view this might be to specify an implementation to use. One of the implementation could be a mock nexmo and another could be Twillio.